GVRf/Framework/framework/src/main/jni/objects/shader_data.cpp

```diff
@@ -240,7 +240,7 @@ namespace gvr {
                 }
             }
         }
-        return (uniforms().updateGPU(renderer) ? 1 : 0);
+        return 1;
```

Comment:
the call is made for updating UBO for material which we are not using right now.